### ISSUE TITLE
Go Get Problems

### ISSUE THREAD

**Issue by cbehopkins** [andlabs/ui#196]
I suspect I am on the incorrect version of GCC? Either that or it's not happy with having my git repository on a drive that isn't c?

```
 go get github.com/andlabs/ui
# github.com/andlabs/ui
P:/git/src/github.com/andlabs/ui/libui_windows_amd64.a(alloc.cpp.obj): In function `uiAlloc':
E:/github.com/andlabs/libui/windows/alloc.cpp:34: undefined reference to `operator new(unsigned long long)'
E:/github.com/andlabs/libui/windows/alloc.cpp:34:(.text+0x237): relocation truncated to fit: R_X86_64_PC32 against undefined symbol `operator new(unsigned long long)'
```

There's a lot more error messages than that but they're all very similar

**Comment by andlabs** [comment]
What is your gcc?

**Comment by Goblinlordx** [comment]
Same issue:
Target: x86_64-pc-cygwin
Version: gcc version 5.4.0 (GCC)

**Comment by comaldave** [comment]
# I seem to have a similar problem.
gcc.exe (tdm64-1) 5.1.0

```bat
    > go get github.com/andlabs/ui
    # github.com/andlabs/ui
    C:/Users/skinn/local/src/github.com/andlabs/ui/libui_windows_amd64.a(stddialogs.cpp.obj): In function
    msgbox':
    E:/github.com/andlabs/libui/windows/stddialogs.cpp:113: undefined reference to `__imp_TaskDialog'
    collect2.exe: error: ld returned 1 exit status
````

**Comment by andlabs** [comment]
@comaldave your problem is an old MinGW. Get MinGW-w64 5.

**Comment by andlabs** [comment]
Merged with #281 and #279 (respectively).
